libbeat/docs/security/users.asciidoc

```diff
@@ -0,0 +1,182 @@
+[role="xpack"]
+[[feature-roles]]
+=== Grant users access to secured resources
```

Comment:
I need to add the apm-server role to this file.

Reply:
@bmorelli25 These changes will require additional testing for APM. I've left out information about APM, but can add it if you want to review this content, or I can merge this without APM info, and you can submit a PR on top. WDYT?

Reply:
I'm thinking it'll be easier if I open a PR after